### DISCUSSION TITLE
Installer instead of Windows Store

### DISCUSSION THREAD

**Discussion by joostiphone** [steam3d/MagicPods-Windows#61] [Q&A] [answered]
Hello!

Is it possible to buy the app and getting the installer (msi/exe) outside of the Windows Store? The company I work for is blocking the Windows Store, but I am able to install applications outside of the Windows Store. 

Thank you,
Kind regards,
Joost

**Comment by steam3d** [comment, marked as answer]
Hi. You can install only from Microsoft Store.

**Reply by joostiphone** [reply]
That's unfortunate. Thank you for your answer tho.

**Reply by someone9201** [reply]
Привет! Есть ли возможность получить exe? Не хочу ставить стор, тем более врядли они сейчас выплачивают:( можно ли как нибудь оплатить и получить? Спасибо!

**Reply by steam3d** [reply]
Добрый день. Приложение все ещё доступно только через Microsoft Store. Пока нет вариантов скачать вне магазина.
